lib/babel.js

```diff
@@ -2,6 +2,8 @@
 var resolveFrom = require('resolve-from');
 var createEspowerPlugin = require('babel-plugin-espower/create');
 var requireFromString = require('require-from-string');
+var hasha = require('hasha');
+var cache = require('cacha')('.ava/cache');
```

Comment:
We are going to need a way to include the settings for the transform in the hash as well.

Perhaps it is:
```
  MD5(MD5(source_file_contents) + MD5(
    ava_version + '\n' +
    settings_file_contents + '\n' +
    ...???...
  ));
```
How do we handle changing dependency versions? What happens when the `babel-plugin-transform-runtime` has a patch level bump (meaning the output of the transform may have changed some for the same input). Are we going to have to scan all the plugin version numbers and make those part of the hash? I can't think of an other way.

However we do it, we should compute the part of the hash that is consistent across the test run once (the `ava_version` + `plugin_hash` part), and pass that to the forked processes so they only need to compute the hash of the main file.